src/component/settings/AccountSettings.jsx

```diff
@@ -17,6 +17,7 @@ const AccountSettings = () => {
     firstName: 'First Name',
     lastName: 'Last Name',
     aboutMe: 'About Me',
+    dietaryPrefs: 'Dietary',
```

Comment:
Should this be 'Dietary Preferences' or is that too long for the mobile version?